app/src/main/java/com/example/neighborly/Item.java

```diff
@@ -1,5 +1,52 @@
 package com.example.neighborly;
 
-public class Item{
+import android.graphics.Bitmap;
 
+import com.google.firebase.auth.FirebaseUser;
+
+public class Item {
+
+    private Bitmap image;
+    private String name;
+    private FirebaseUser owner;
```

Comment:
Instead, this can be the user ID, that is generated by Firebase

Reply:
so string?